ci/cases/pr/C48_S2SW_extended.yaml

```
@@ -21,4 +21,3 @@ skip_ci_on_hosts:
   - orion
   - hercules
   - awsepicglobalworkflow
-  - wcoss2  # TODO run on WCOSS2 once the gfs_waveawipsbulls job is fixed
```

This can be reverted.

```suggestion
  - wcoss2  # TODO run on WCOSS2 once the gfs_waveawipsbulls job is fixed
```